src/components/ModalSearch/index.js

```diff
@@ -40,11 +41,16 @@ export const FacetModal = props => {
       closeTimeoutMS={200} >
       <div className='modal-header--search'>
         <h2 className='modal-header__title' aria-live='polite' aria-atomic='true'>
-          {`Filter ${props.resultsCount} Search ${props.resultsCount === 1 ? 'Result': 'Results'}`}
+          <Trans comment='Filter results message'>
+            <Plural value={props.resultsCount} one="Filter # Search Result" other="Filter # Results"/>
+          </Trans>
```

Comment:
The Trans macro is around the Plural macro to easily assign a comment to the plural. Additionally, the `#` will be replaced with the value after the translation.